### DISCUSSION TITLE
How do I get local search to index dynamic pages?

### DISCUSSION THREAD

**Discussion by eagereyes** [vuejs/vitepress#2666] [Q&A]
I'm starting to move a site to vitepress that has some dynamic pages. When I turn on local search, it indexes all of the static pages just fine, but not the dynamic ones.

Do I have to give it an index of those pages somewhere? They're all reachable from the sidebar navigation.

Also, is there a way to see what has been indexed?

**Comment by sbergo** [comment]
I have the same problem.
